internal/gatewayapi/contexts.go

```diff
@@ -165,18 +165,18 @@ type RouteContext interface {
 	// GetRouteStatus returns the RouteStatus object associated with the Route.
 	GetRouteStatus() *v1beta1.RouteStatus
 
-	// TODO: [v1alpha2-v1beta1] This should not be required once all Route
-	// objects being implemented are of type v1beta1.
 	// GetParentReferences returns the ParentReference of the Route object.
+	// objects being implemented are of type v1beta1.
+	// TODO: [v1alpha2-v1beta1] This should not be required once all Route
 	GetParentReferences() []v1beta1.ParentReference
 
 	// GetRouteParentContext returns RouteParentContext by using the Route
 	// objects' ParentReference.
 	GetRouteParentContext(forParentRef v1beta1.ParentReference) *RouteParentContext
 
-	// TODO: [v1alpha2-v1beta1] This should not be required once all Route
-	// objects being implemented are of type v1beta1.
 	// GetHostnames returns the hosts targeted by the Route object.
+	// objects being implemented are of type v1beta1.
```

Comment:
needs to be reverted

Reply:
![image](https://user-images.githubusercontent.com/2174082/222035632-ba0576f4-41cf-464f-80d5-3b7556d1d4fe.png)

catch this warning